src/module-2/bootstrap-design/index.html

```diff
@@ -0,0 +1,618 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Main</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600&display=swap"
+        rel="stylesheet">
+  <link crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css"
+        integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" rel="stylesheet">
+  <script crossorigin="anonymous"
+          integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ"
+          src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body class="container-fluid bg-light">
+<header class="mb-4 row">
+  <div class=" d-flex flex-row align-items-center justify-content-start logo">
+    <img alt="Logo" class="m-4" src="images/logo.svg">
+    <h1 class="text-primary">Online Store</h1>
+  </div>
+  <nav>
+    <a class=" mx-2 text-decoration-none" href="">
+      <svg class="bi bi-house" fill="currentColor" height="16" viewBox="0 0 16 16" width="16"
```

Comment:
minor: you may also download corresponding `svg` files from bootstrap page and store them in some `icons` folder with meaningful names, that would be a cleaner and more readable solution

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)